### ISSUE TITLE
[Bug] Present news infobar is showing 

### ISSUE THREAD

**Issue by Harmonie666** [YTVanced/Vanced#234]
**Bug description**
Since a few days now, I'm seeing the news bar again. It wasn't showing before, but now pops up again very frequently. I think the setting that is supposed to turn it off is broken now. 

**Variant**
NonRoot: Yes
Root: No

**Vanced version**
15.43.32 - Build - 01.47.00

**Android version**
Android 10

**Device**
ASUS ROG Phone II

**Steps to Reproduce**
Steps to reproduce the error:
1. Open the app
2. If necessary scroll down
3. You should see the present news infobar

**Expected behavior**
Please remove it from Vanced. Those infobars are annoying. Mostly about Covid-19 plandemic and are taking up unnecessary space.
![Screenshot_20201127-222537602 (1)](https://user-images.githubusercontent.com/73917917/100485529-b4528500-3100-11eb-934d-c4289e17ffb2.png)

**Comment by Harmonie666** [comment]
@VancedOfficial 
Are you devs going to remove the annoying news/vaccine propaganda?
It appears to many times, and dismissing it doesn't help a bit. 
It's getting really annoying:
![Screenshot_20210706-213009507 (1).png](https://user-images.githubusercontent.com/73917917/124656549-a4263380-dea1-11eb-922e-808ed982e83d.png)

**Comment by umbrellacorp53** [comment]
I see that this is marked as "won't fix"... Look, I know you don't want to be "on the same side" as the wacky anti-vaxxers. That's not what is being requested of you, and not what this toggle would represent. 

I have family who are dead and in ICU and medically induced coma from covid. I have been vaccinated and I am educated on the current state of things. So is everyone I know and I try to convince my folks to get vaccinated. 

I get on YouTube to choose what content I see. I installed vanced to allow me to choose who I support and by what means. I would like the choice to turn off invasive news stories that seem to pop up when I'm not prepared. I do not want to think about my cold dead uncle, the funeral, my elderly parents who refuse the vaccine, my only uncle who is left but in ICU, the children in hospice that I passed on my way to see him, or the terrifying supply chain issues, food shortages, the economic issues, new variants, etc. when I get on this app. 

Sure, it should be left alone by default so your work does not have a negative impact, but for those of us who are trying to learn and stay focused, need to escape, or are dealing with grief, anxiety, depression, etc. we should not be FORCED to see these reminders of the horrors we're living through, especially at random, without consent, and with no warning.

The only people who would choose to turn this off are either looking for solice from the reminders of tragedy that are seemingly inescapable and non-negotiable, or are the types who are entrenched into a political ideology and are unwilling to listen and therefore the message is lost on them anyway... So choosing to ignore this issue just to spite people who will not change is only hurting the former who frankly could use a brief escape. Either way, not giving the choice, when it is clearly within your power is not helpful.
